packages/models/src/lib/notifications/mark-read-request.model.ts

```diff
@@ -0,0 +1,3 @@
+export interface MarkReadRequest {
```

Comment:
This filename doesn't really follow our naming conventions, but just calling it `MarkRead` is kind of vague. Thoughts?